arbitrum-docs/run-arbitrum-node/sequencer/high-availability-sequencer-docs.mdx

```diff
@@ -0,0 +1,479 @@
+
+If your sequencer goes offline, the network will be unable to process new transactions that come to child chain RPC nodes, effectively affecting the user experience. This guide provides detailed instructions for setting up a high availability (HA) sequencer architecture to minimize downtime and ensure your Arbitrum chain remains operational even if individual components fail.
+
+## Prerequisites
+
+Before you begin, ensure you have:
+
+- Experience with Kubernetes and container orchestration
+- Access to a Kubernetes cluster with multiple availability zones
+- Understanding of Redis and cloud infrastructure
+- A properly configured parent chain node or RPC node endpoint
+- Sequencer keys and appropriate permissions
+- Sufficient storage and compute resources
+
+## High Availability Sequencer Architecture
+
+A high availability sequencer deployment consists of seven key components:
+
+1. **CDN/Load Balancer** - Manages traffic routing and bot detection
+2. **Nitro Fullnodes** - Process read requests and forward write transactions
+3. **External Relays** - Handle public feed traffic
+4. **Sequencer Relays** - Combine feeds from all sequencers
+5. **Sequencers** - Multiple redundant transaction queueing machines
+6. **Redis** - Coordinates active sequencer selection and sequencer-related information sharing
+7. **Batch Poster** - Posts transaction batches to the parent chain
+
+### Architecture Diagrams
+
+The architecture varies slightly depending on whether your full nodes use Redis to identify the active sequencer or forward transactions to a predefined endpoint.
+
+#### Send to Active Enabled
+
+In this configuration, full nodes query Redis to determine the active sequencer and forward transactions directly to it:
+
+<ImageZoom
+  src="/img/ha-sequencer-sendto-active.png"
+  alt="send to active enable"
+  className="img-600px"
+/>
+
+#### Send to Active Disabled
+
+In this configuration, full nodes forward transactions to a predefined endpoint without checking which sequencer is active:
+
+<ImageZoom
+  src="/img/ha-sequencer-sendto-nonactive.png"
+  alt="send to active disable"
+  className="img-600px"
+/>
+## Detailed Component Setup
+
+### 1. Load Balancing (CDN)
+
+It is a strong recommendation to use a CDN for managing traffic and security concerns:
+
+- **Recommendation**: Use Cloudflare or a similar CDN service
+- **Configuration**:
+  - Direct `/rpc` traffic to the Nitro full node fleet
+  - Direct `/feed` traffic to public-facing relays
+  - Implement rate limiting and bot detection as needed
+- **Benefits**: Distributes load, improves security, and enhances availability
+
+### 2. Relays Setup
+
+The requirement is for two types of relays in the architecture:
+
+#### Sequencer Relays
+
+- Deployment should have multiple replicas
+- Configure to listen to feed outputs from all sequencers
+- All other components connect to these relays instead of directly to sequencers
+- Minimize direct load on sequencer nodes
```

Comment:
```suggestion
- Deployment should have multiple replicas
- Configure to listen to feed outputs from all sequencers
- All other components connect to these relays instead of directly to the sequencers
- Minimize direct load on sequencer nodes
```